ios/engine/KMEI/KeymanEngine/Classes/LanguagePicker/ResourceInfoView.xib

```diff
@@ -0,0 +1,107 @@
+                    <rect key="frame" x="0.0" y="0.0" width="320" height="568"/>
+                    <subviews>
+                        <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="OZ0-0W-ym5" userLabel="Content View">
+                            <rect key="frame" x="0.0" y="0.0" width="320" height="568"/>
+                            <subviews>
+                                <tableView clipsSubviews="YES" contentMode="scaleToFill" alwaysBounceVertical="YES" scrollEnabled="NO" style="plain" separatorStyle="default" rowHeight="-1" estimatedRowHeight="-1" sectionHeaderHeight="28" sectionFooterHeight="28" translatesAutoresizingMaskIntoConstraints="NO" id="e0s-9L-REf">
+                                    <rect key="frame" x="0.0" y="0.0" width="320" height="184"/>
+                                    <color key="backgroundColor" systemColor="systemBackgroundColor" cocoaTouchSystemColor="whiteColor"/>
+                                    <constraints>
+                                        <constraint firstAttribute="height" constant="184" id="iOy-df-nx4"/>
+                                    </constraints>
+                                </tableView>
+                                <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="wMn-do-Ek6" userLabel="QR View">
+                                    <rect key="frame" x="0.0" y="184" width="320" height="384"/>
+                                    <subviews>
+                                        <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleAspectFit" horizontalHuggingPriority="251" verticalHuggingPriority="251" translatesAutoresizingMaskIntoConstraints="NO" id="ysW-nc-xvg">
+                                            <rect key="frame" x="72.5" y="20" width="175" height="175"/>
+                                            <constraints>
+                                                <constraint firstAttribute="height" constant="175" id="zC7-D3-1LZ"/>
+                                                <constraint firstAttribute="width" secondItem="ysW-nc-xvg" secondAttribute="height" multiplier="1:1" id="zHh-Ye-Nlf"/>
+                                            </constraints>
+                                        </imageView>
+                                        <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="301" verticalHuggingPriority="301" horizontalCompressionResistancePriority="800" verticalCompressionResistancePriority="800" text="Scan this code to load this resource on another device" textAlignment="center" lineBreakMode="wordWrap" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="z2O-MT-IoV">
```

Comment:
Rather than 'load this resource' we should say 'load this keyboard' or 'load this dictionary' -- some users are unlikely to know what a 'resource' is in this context.